### PR TITLE
GDB cheat sheet: Added two commands

### PR DESCRIPTION
**Pull request by mayank7**
Added two commands to GDB cheat sheet.

IA Page: https://duck.co/ia/view/gdb_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [GDB Cheat Sheet](https://duck.co/ia/view/gdb_cheat_sheet)

**Description**: Overview of essential GDB commands

**Example Query**: [GDB Cheat Sheet](https://beta.duckduckgo.com/?q=GDB Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/gdb_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/gdb_cheat_sheet)*

File: share/goodie/cheat_sheets/json/gdb.json [hunk 1]
```diff
@@ -32,6 +32,14 @@
                 "key": "gdb \\[program\\]"
             },
             {
+                "val": "Begin debugging program and pass argument(s)",
+                "key": "gdb --args \\[program\\] \\[argument(s)\\]"
+            },
+            {
+                "val": "Begin debugging program and attach to process",
+                "key": "gdb --pid \\[process\\]"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/gdb.json]: Shouldn't there be a `\\[program\\]` in here?

Comment (mayank7) [inline share/goodie/cheat_sheets/json/gdb.json]: @GuiltyDolphin I changed it. Now is it okay ??

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**